pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -39,6 +39,22 @@ import (
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
 )
 
+var testParams = map[string]struct {
```

Comment:
Why is this struct needed? Is it for some future use? Currently, IMO, it doesn't improve readably.

Reply:
Hey I answer for this [here](https://github.com/kubevirt/kubevirt/pull/3328/files/33ba0b15fdcb4c1aff850d50e7b7ac60418b5db4#r416415669)

Reply:
I removed this map, now there is a struct with a property for each address/cidr